.clj-kondo/config.edn

```diff
@@ -652,7 +653,7 @@
    :name    test-namespaces}
   {:pattern "^(?!.*-test(?:\\..*)?$)(?!.*test-util$)(?!^metabase\\.test.*).*$"
    :name    source-namespaces}
-  {:pattern "metabase\\.cmd.*"
+  {:pattern "(?:(?:^metabase\\.cmd.*)|(?:^build.*)|(?:^metabuild-common.*)|(?:^release.*)|(?:^i18n.*)|(?:^lint-migrations-file$))"
```

Comment:
Thanks to Cam Saul [for this pattern](https://github.com/metabase/metabase/commit/90b876c847d782a9b64d662de0e676c41154e0a8#diff-d8bfac7f9aa64d7a63a2c6810095e91e716527266288a210b3f31a2c141ce3f2R621), and about half of this PR.